### PR TITLE
Fixed ppn hint and jobID capture on mike

### LINKED ISSUE #1306: Correct `--ntasks-per-node` on mike
I think they have made a configuration change ... in the `padcirc.slurm` file for the Wind10m scenario, which normally runs on 16 cores, the `--ntasks-per-node` was set to 64 (which apparently used to work) but now it needs to be set the same as `--ntasks` ... I will be logging an issue for this, fixing the `HPC_PPN_Hints()` function in `platforms.sh` and restarting it...

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #1306 with additional testing now

### CODE REVIEW

**Comment by wwlwpd** [comment]
Looking good, I'll keep an eye out for you making this ready for review.

**Comment by jasonfleming** [comment]
Tested successfully. This is ready to review and merge.

File: asgs_main.sh [hunk 1]
```diff
@@ -1333,9 +1333,9 @@ submitJob()
    echo "hpc.job.${JOBTYPE}.cloptions : \"$CLOPTIONS\"" >> $ADVISDIR/$ENSTORM/run.properties
    echo "hpc.job.${JOBTYPE}.localhotstart : $LOCALHOTSTART" >> $ADVISDIR/$ENSTORM/run.properties
    _CPUREQUEST=$(($NCPU + $NUMWRITERS))
-   _PPN=$(HPC_PPN_Hint "parallel" "$QUEUENAME" "$HPCENV" "$QOS" "$PPN" )
-   _RESERVATION=$(HPC_Reservation_Hint "$RESERVATION" "$HPCENV" "$QOS" "$CPUREQUEST")
    _QUEUENAME=$(HPC_Queue_Hint "$QUEUENAME" "$HPCENV" "$QOS" "$CPUREQUEST")
+   _PPN=$(HPC_PPN_Hint "parallel" "$QUEUENAME" "$HPCENV" "$QOS" "$PPN" "$CPUREQUEST")
+   _RESERVATION=$(HPC_Reservation_Hint "$RESERVATION" "$HPCENV" "$QOS" "$CPUREQUEST")
```
Comment (wwlwpd) [inline asgs_main.sh:1338]: Looks like you found and fixed an issue with this call to `HPC_Reservation_Hint`, too. Good catch. We should just be aware of anything that might have been relying on this bug (no reason to not approve). Still a good catch and fix.

**Review by wwlwpd — APPROVED** [review]
looks good to me!

**Pull request merged**